apps/docs/src/app/platform/component-docs/platform-forms/radio-group/platform-radio-group-docs.component.html

```diff
@@ -1,44 +1,48 @@
+    Radio Button Group With Radio Buttons Passed as Content.
 </fd-docs-section-title>
 
-<description>This example shows Radio Button Group implementaion with Content as Radio Buttons</description>
+<description>
+    Platform Radio Buttons Passed as Content
+</description>
 <component-example>
     <fdp-platform-radio-group-content-example></fdp-platform-radio-group-content-example>
 </component-example>
 <code-example [exampleFiles]="contentRadioGroup"></code-example>
 
-<fd-docs-section-title [id]="'def'" [componentName]="'Disabled Radio Button Group'">
-    Examples for Radio Button Group with List of values
+<fd-docs-section-title [id]="'list'" [componentName]="'Disabled Radio Button Group'">
+    Radio Button Group Creates Radio Buttons With List of String Values.
 </fd-docs-section-title>
 <description>
-    This example shows Regular Radio Button Group implementaion with list of string Values.<br />
+    Creating Platform Radio Buttons with List of String Values.<br />
     eg. ['Summer', 'Winter', 'Autumn'], Radio buttons are created with these values.
 </description>
 <component-example>
     <fdp-platform-radio-group-list-example></fdp-platform-radio-group-list-example>
 </component-example>
 <code-example [exampleFiles]="listRadioGroup"></code-example>
 
-<fd-docs-section-title [id]="'def'" [componentName]="'Radio Button Group'">
-    Examples for Radio Button Group with List of objects (SelectItem Object)
+<fd-docs-section-title [id]="'listobject'" [componentName]="'Radio Button Group'">
+    Radio Button Group Creates Platform Radio Buttons With List of Objects. <br />
 </fd-docs-section-title>
 
 <description>
-    This example shows Regular Radio Button Group implementaion with list of Objects of SelectItem.<br />
-    Radio buttons are created with these values.
+    Creating Platform Radio Buttons with List of Object.<br />
+    Here value, label and disabled properties of SelectItem object or LookupKey and DisaplayKey of any objects are used
+    to Create Individual Radio Buttons.
 </description>
 <component-example>
     <fdp-platform-radio-group-list-items-example></fdp-platform-radio-group-list-items-example>
 </component-example>
 <code-example [exampleFiles]="listItemsRadioGroup"></code-example>
 
-<fd-docs-section-title [id]="'def'" [componentName]="'Disabled Radio Button Group'">
-    Examples for Disable Radio Button Group
+<fd-docs-section-title [id]="'disabled'" [componentName]="'Disabled Radio Button Group'">
+    Radio Button Group for Disable Radio Buttons. Here One or More Radio Buttons Can be Disabled and Others Can be
+    Enabled State.
 </fd-docs-section-title>
-<description
-    >This example shows disabled radio buttons, passed as content or list of values. <br />
-    It covers Reactive form approach as well as Template driven form
+<description>
+    Creation of disabled radio buttons. This can be done when radio buttons passed as content or created with list of
```

Comment:
I'll be able to help with this once I understand what "passed as content" means. but it might be better to title it as "Creating disabled radio buttons is possible when radio buttons are _passed as content_ or created with a list of values."

Reply:
corrected.
corrected to "passed as content projection". It means that some content going under some other html element.
Ex:
`<radio-group> 
       <radio>radio1</radio> 
       <radio>radio2</radio> 
</radio-group>`